test/test_simulation_build.jl

```diff
@@ -1,13 +1,15 @@
-path = joinpath(pwd(), "test_sequence_build")
-!isdir(path) && mkdir(path)
+g_test_path = joinpath(pwd(), "test_sequence_build")
+!isdir(g_test_path) && mkdir(g_test_path)
 
-function test_sequence_build(file_path::String)
-    stages_definition = Dict(
-        "UC" => Stage(GenericOpProblem, template_basic_uc, c_sys5_uc, GLPK_optimizer),
+function create_stages(template_uc)
+    return Dict(
+        "UC" => Stage(GenericOpProblem, template_uc, c_sys5_uc, GLPK_optimizer),
```

Comment:
can we restore this to `template_basic_uc`?

Reply:
This function is called many times in this file.  Two different global variables get passed in:  `template_basic_uc` and `template_standard_uc`.

I'm OK with reverting all changes in this file.  They are not needed because Stage will support `reset!`.  I left my changes here to let you see what the non-reset workflow looks like.  Let me know.